apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -3316,19 +3355,21 @@ sc_ws_withdraw(Config) ->
 %%     channel_options(IPubkey, RPubkey, IAmt, RAmt, #{}).
 
 channel_options(IPubkey, RPubkey, IAmt, RAmt, Other, Config) ->
-    maybe_slogan(
-      maybe_add_password(
-        maps:merge(#{ port => ?config(ws_port, Config),
-                      initiator_id => aeser_api_encoder:encode(account_pubkey, IPubkey),
-                      responder_id => aeser_api_encoder:encode(account_pubkey, RPubkey),
-                      lock_period => 10,
-                      push_amount => 1,
-                      initiator_amount => IAmt,
-                      responder_amount => RAmt,
-                      channel_reserve => 2,
-                      keep_running => false,
-                      protocol => sc_ws_protocol(Config)
-                    }, Other), Config), Config).
+  %% TODO FIX THOSE MAYBES
+    maybe_fee(
```

Comment:
I guess these could be generalized to something like `maybe_set([slogan, fee], maps:merge(...))`, but the benefit would be marginal.

Reply:
@velzevur You could use a similar approach as I've added here: https://github.com/aeternity/aeternity/blob/master/apps/aechannel/test/aesc_fsm_SUITE.erl#L3821

Reply:
I honestly forgot about this TODO :)